modular_skyrat/modules/GAGS/greyscale_configs.dm

```diff
@@ -106,6 +118,11 @@
 	name = "Ranger Poncho Worn"
 	json_config = 'modular_skyrat/modules/GAGS/json_configs/ranger_poncho/ranger_poncho_worn.json'
 
+/datum/greyscale_config/ranger_poncho/teshari
```

Comment:
make sure this is a subtype of ranger_poncho/worn

Reply:
you also wont need a new json_config if you use the same icon states btw, only the icon_file